src/components/Thanks/ThanksLayoutV2.vue

```diff
@@ -168,6 +199,47 @@
 						</div>
 					</kv-expandable>
 				</div>
+				<!-- auto deposit (ad) -->
+				<div
+					class="kv-accordion thanks-page__content-area thanks-page__content-area--ad"
+					:class="{
+						'kv-accordion--open' : isAdSelected,
+					}"
+					v-if="showAutoDepositUpsell"
+				>
+					<kv-icon-button class="thanks-page__icon-button expanded"
+						:class="{ active: isAdSelected }"
+						aria-controls="`kv-accordion-mg-accordion`"
+						:aria-expanded="isAdSelected ? 'true' : 'false'"
+						@click.native="setVisibleSection('ad')"
+					>
+						<template #icon-left>
+							<kv-icon
+								name="alert-circle"
+							/>
+						</template>
+						Auto-deposit
+						<template #icon-right>
+							<kv-icon
+								class="tw-w-2.5 tw-h-2.5"
+								name="fat-chevron"
+								:from-sprite="true"
+							/>
+						</template>
+					</kv-icon-button>
+					<kv-expandable>
+						<div
+							class="kv-accordion__pane"
+							id="kv-accordion-ad-accordion"
+							v-show="isAdSelected"
+							:aria-hidden="isAdSelected ? 'false' : 'true'"
+						>
+							<hr>
+							<slot v-if="isMobile" name="ad">
```

Comment:
These `isMobile` checks allow us to selectively render the braintree signup form based on viewport so there are not two instances in the page at once, as one of them would always throw an error.